vizro-core/src/vizro/actions/export_data_class_action.py

```diff
@@ -0,0 +1,110 @@
+import importlib
+from typing import Any, Dict, List, Literal
+
+from dash import Output, State, ctx, dcc
+
+from vizro.actions._filter_action import _filter
+from vizro.actions.filter_interaction_action import filter_interaction
+from vizro.managers import model_manager
+from vizro.models.types import CapturedActionCallable
+
+
+class ExportDataClassAction(CapturedActionCallable):
+    def __init__(self, *args, **kwargs):
```

Comment:
What's this `__init__` for?

Reply:
I used it to save input args and kwargs so they can be validated/adjusted in the `_post_init`.
Now, I got rid of the constructor and `self._arguments` is used inside the `_post_init`.